app/views/activities/_poll_methodologies.js.erb

```diff
@@ -0,0 +1,51 @@
+<%# Card activities %>
```

Comment:
We are porting this, but never calling this partial. 
It should be from `activities/poll.js.erb`
But I think this belongs to another PR.